dao/src/main/java/greencity/entity/user/Region.java

```diff
@@ -1,42 +1,41 @@
 package greencity.entity.user;
 
-import jakarta.persistence.CascadeType;
-import jakarta.persistence.Column;
+import greencity.entity.user.locations.BaseEntityForEnAndUkNames;
+import greencity.entity.user.locations.City;
 import jakarta.persistence.Entity;
-import jakarta.persistence.Table;
-import jakarta.persistence.Id;
 import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.OneToMany;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.FetchType;
 import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 import lombok.ToString;
 import java.util.List;
+import lombok.experimental.SuperBuilder;
 
 @NoArgsConstructor
 @AllArgsConstructor
 @Getter
 @Setter
-@Builder
+@SuperBuilder
 @Table(name = "regions")
 @EqualsAndHashCode(exclude = {"locations"})
-@ToString(exclude = {"locations"})
+@ToString(exclude = {"locations", "cities"})
 @Entity
-public class Region {
+public class Region extends BaseEntityForEnAndUkNames {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "region")
     private List<Location> locations;
 
-    @Column(name = "name_uk")
-    private String ukrName;
-
-    @Column(name = "name_en")
-    private String enName;
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "region", fetch = FetchType.LAZY)
```

Comment:
Are you sure we need `CascadeType.ALL` here?

Reply:
you are right, in this case will be a good to change it to PERSIST type